sdk/core/core-util/test/public/bytesEncoding.spec.ts

```diff
@@ -1,150 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-import { stringToUint8Array, uint8ArrayToString } from "../../src/bytesEncoding.js";
-import { describe, it, assert } from "vitest";
```

Comment:
Totally fine with removing these, but just to confirm - did you get a chance to run these tests once and make sure they still pass after moving things over? Once this is merged, they add less value I agree, but while we're refactoring the tests help ensure we didn't regress.

I'm fine with either "I ran them once and they all pass" or with "I'll put them back, and remove them in a follow-up PR" - whatever you think makes more sense